app/views/partials/channels-tab-request-matching.html

```diff
@@ -1,8 +1,12 @@
-<div class="form-group-2columns" style="margin-bottom: 15px;">
-  <div class="form-group" style="float: left; width: 75%; margin-bottom: 0px;" ng-class="{ 'has-error' : ngError.urlPattern }" ng-if="channel.type === 'http' || channel.type === 'polling'">
+<p>
+  Here the requests that will be forwarded to this channel can be configured.
```

Comment:
I would suggest: _Configure the matching criteria for requests that should be forwarded to this channel_